pyproject.toml

```diff
@@ -529,6 +529,7 @@ python_files = [
 ]
 testpaths = [
     "tests",
+    "task_sdk/tests",
```

Comment:
Not sure about this one. (I didn't seem to have to do it for provider tests.)

Reply:
Yeah `ruff` was detecting usage of "assert" in test file and failing: https://docs.astral.sh/ruff/rules/assert/

I was trying to figure it out what to do and then saw that you already ignored S101: https://github.com/apache/airflow/blob/main/providers/pyproject.toml#L57 so I copied it

So, yeah L532 here can be removed.